particles/MeshParticles.hpp

```diff
@@ -14,25 +14,20 @@ namespace particles
 /** Here we define a mesh fairly manually and in a fairly suboptimal way
  * (based on this: https://pastebin.com/DXKEmvap)
  */
-struct TexturedMeshForParticles final : score::gfx::Mesh
+struct TexturedMeshForParticles
```

Comment:
Removed the inheritance from the score class which was making things more complicated